config/rbac/role.yaml

```diff
@@ -177,6 +177,7 @@ rules:
   - machineconfigs
   verbs:
   - list
+  - watch
```

Comment:
I am seeing this again

> I was getting errors in the WMCO log around failing to get the machine config resource because it was unable to watch it. I didn't really understand why I was getting the error honestly, but giving watch permissions "fixed" it.